src/server/index.test.jsx

```diff
@@ -44,22 +53,6 @@ styledComponents.ServerStyleSheet = jest.fn().mockImplementation(() => ({
 describe('Server', () => {
   const makeRequest = async path => request(server).get(path);
 
-  it(`should not pass an 'x-powered-by' response header`, async () => {
-    const { headers } = await makeRequest('/status');
-    const headerKeys = Object.keys(headers);
-    expect(headerKeys).not.toContain('x-powered-by');
-  });
-
-  it(`should set 'x-frame-options' header to 'DENY'`, async () => {
-    const { headers } = await makeRequest('/status');
-    const headerKeys = Object.keys(headers);
-    const headerValues = Object.values(headers);
-    const indexOfXFrame = headerKeys.indexOf('x-frame-options');
-
-    expect(headerKeys).toContain('x-frame-options');
-    expect(headerValues[indexOfXFrame]).toEqual('DENY');
-  });
-
   describe("should set 'x-clacks-overhead' header", () => {
```

Comment:
Can this header be moved under the [header describe block](https://github.com/BBC-News/simorgh/pull/799/files#diff-9d7ac6551ba249f8b551e1ca9306caa4R168)?

Reply:
This still seems relevant